finalPrep/modZeroHero.js

```diff
@@ -2,45 +2,97 @@
 
+var heroName = 'Red Rush';
+
+var specialAbility = 'Super Speed';
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "There is nothing to fear, " + heroName + " is here!";
+
+var catchphrase = `My ${specialAbility} is all we NEED!`;
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 50;
+
+var energy = 1000000;
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
+
+var fullPower = power * 500;
+
 //   fullEnergy should add 150 to your current energy
 
+var fullEnergy = energy + 150;
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = true;
+
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
+
+var archEnemies = ['Slow Blue', 'Cool Rush', 'Micheal Keaton'];
+
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var sidekicks = ['Green Rush', 'Backpack Kid', 'Rush Jr.'];
+
 // Print the first sidekick to your console
 
+console.log(sidekicks[0]);
+
 // Print the last archEnemy to the console
 
+console.log(sidekicks[2]);
+
 // Write some code to add a new archEnemy to the archEnemies array
 
+archEnemies.push('Young Neil');
+
 // Print the archEnemies array to console to ensure you added a new archEnemy
 
+console.log(archEnemies);
+
 // Remove the first sidekick from the sidekicks array
 
+sidekicks.shift();
+
 // Print the sidekicks array to console to ensure you added a new sidekick
 
+console.log(sidekicks);
+
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
 
 // Your function should include an if/else statement that meets the following criteria
 //   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
 //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
 //   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
 
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+    if (dangerLevel > 50) {
+      console.log(badExcuse);
+    } else if (dangerLevel >= 10 && dangerLevel < 50) {
+          console.log(saveTheDAy);
+      } else {
```

Comment:
Your else block should be the same indentation as your else if. Also, you can think about simplifying the logic by considering switching your else and else if blocks.